### ISSUE TITLE
为什么MMaction2中视频分类的得分是负数？

### ISSUE THREAD

**Issue by yinghaodang** [open-mmlab/mmaction2#2063]
我正在学习MMAction2 Tutorial。
当我在尝试这一步的时候：用MMAction2的识别模型做一次推理
我在源码 [mmaction/api/inference.py] 中增加了两行输出
<img width="581" alt="4dd54a1d8205d9371215b8107e05fb0" src="https://user-images.githubusercontent.com/49315320/202408764-7dcf1b0e-8cd9-44ed-8da3-8f5fbf580dcc.png">
我仅仅是想看一下各个分类的得分，以及得分的总和。
于是我得到了如下的结果：
<img width="646" alt="d0bbb45bae5276c1abd41dc33ae983a" src="https://user-images.githubusercontent.com/49315320/202411808-06b26979-cfe4-44b2-a7bf-1a90078b2395.png">
<img width="615" alt="462f269a58c61af62e4da417182e229" src="https://user-images.githubusercontent.com/49315320/202409464-07105598-02f4-4bd2-81cd-e098c27c5011.png">
我不知道，这其中的负数是什么意思？我本来预期的是400个类的总和得分是100，但是我得到的是-4.

我会发现这个问题是由于我在使用我自己的数据集进行视频分类的时候，发现结果不是很理想，有时候最高的得分还是负数的。我不知道怎么去理解这个数字。
顺便问一句，我的配置文件采用的是kinetics400的配置文件，调用的是tsn的算法。我要怎么修改我的数据集，才能使得我训练的结果更加好？

**Comment by yinghaodang** [comment]
Do I need to write a new in English？

**Comment by hukkai** [comment]
@yinghaodang I think that is the logits, not the softmax scores.
